README.md

```diff
@@ -191,6 +190,8 @@ This project is licensed under the [Apache License, Version 2.0](https://github.
 
 ## Acknowledgments
 
+[![SonarCloud](https://sonarcloud.io/images/project_badges/sonarcloud-white.svg)](https://sonarcloud.io/dashboard?id=bcgov_ppr)
```

Comment:
I believe dashboard id should be ours right?

Reply:
Oh snap - thanks.